core/server/worker/src/main/java/alluxio/worker/block/DefaultBlockWorker.java

```diff
@@ -296,6 +306,12 @@ public String createBlock(long sessionId, long blockId, int tier,
           sessionId, blockId, tier,
           createBlockOptions.getMedium(), createBlockOptions.getInitialBytes(), e);
 
+      // mAddress is null if the worker is not started
+      if (mAddress == null) {
+        throw new WorkerOutOfSpaceException(ExceptionMessage.CANNOT_REQUEST_SPACE
+                .getMessage(mWorkerId.get(), blockId), e);
+      }
+
```

Comment:
This change is to unblock unit tests. `mAddress` is null until `DefaultBlockWorker` is started, whereas we don't start the worker in unit test